opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -208,4 +208,11 @@ message LogRecord {
   //   - the field is not present,
   //   - the field contains an invalid value.
   bytes span_id = 10;
+
+  // A globally unique identifier of event category/type among all events. All
```

Comment:
We didn't put "globally" for trace id, do we want it here (and what does it mean - do we enforce it?)

Reply:
Removed "globally", added recommendation to keep it "fully qualified"